skorch/callbacks/lr_scheduler.py

```diff
@@ -149,7 +149,14 @@ def on_epoch_end(self, net, **kwargs):
             if callable(self.monitor):
                 score = self.monitor(net)
             else:
-                score = net.history[-1, self.monitor]
+                try:
+                    score = net.history[-1, self.monitor]
+                except KeyError:
+                    raise ValueError(
+                        f"'{self.monitor}' was not found in history. A "
+                        f"PassthroughScoring callback with name='{self.monitor}' "
```

Comment:
```suggestion
                        f"Scoring callback with name='{self.monitor}' "
```

I would suggest to be more agnostic about which kind of scoring is used, since `EpochScoring` would work as well.